charts/telepresence/values.yaml

```diff
@@ -89,6 +89,16 @@ podCIDRs: []
 # Default: auto
 podCIDRStrategy: auto
 
+# dnsServiceName adds a user supplied service to the list of services Telepresence tries to resolve in order to detect the 
+# clusterIP DNS is using
+dnsServiceName: "coredns"
+
+# dnsServiceNamespace specifies what namespace Telepresensce should look for when using the user supplied dnsServiceName
+dnsServiceNamespace: "kube-system"
+
+# dnsServiceIP specifies a fallback clusterIP for Telepresence to use should auto DNS discovery fail, this value is ignored if Telepresence finds a valid clusterIP
+dnsServiceIP: ""
+
```

Comment:
Those additions need must have a corresponding entry in the `charts/telepresence/README.md` file.